content/en/news/releases/1.0.x/announcing-1.0/index.md

```diff
@@ -121,7 +121,7 @@ be configured using [authentication policies](/docs/concepts/security/#authentic
 
 ### `istioctl`
 
-- Added the [`istioctl authn tls-check`](/docs/reference/commands/istioctl/#istioctl-authn-tls-check) command.
+- Added the [`istioctl authn tls-check`](/docs/reference/commands/istioctl) command.
```

Comment:
```suggestion
- Added the [`istioctl authn tls-check`](https://archive.istio.io/v1.0/docs/reference/commands/istioctl/#istioctl-authn-tls-check) command.
```

Reply:
@ericvn Use this same pattern for all broken links in release notes.

Reply:
Done